package.json

```diff
@@ -35,4 +42,4 @@
 	"lint-staged": {
 		"*.{ts,js}": "eslint --cache --fix"
 	}
-}
+}
```

Comment:
All the changes in this file are good, i don't know why it worked for me and not you but this setup is good too, more similar to the setup I usually do at work. 👍